packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -2,40 +2,24 @@ import React from 'react';
 import styled, { css } from 'styled-components';
 import { string, oneOf, bool, node } from 'prop-types';
 import { C_WHITE, C_EBON } from '@bbc/psammead-styles/colours';
-import {
-  GEL_SPACING_HLF,
-  GEL_SPACING,
-  GEL_SPACING_QUAD,
-} from '@bbc/gel-foundations/spacings';
-import { GEL_GROUP_1_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/breakpoints';
-import { GEL_MINION } from '@bbc/gel-foundations/typography';
+import { GEL_SPACING_HLF } from '@bbc/gel-foundations/spacings';
```

Comment:
Seems like we still are using this GEL_SPACING_HLF,

When looking at where it's used, it's applied to the Time element
https://github.com/bbc/psammead/blob/302a5857b799b4f2cd192a0ccfe468ba5bd86487/packages/components/psammead-media-indicator/src/index.jsx#L33

Makes me think that this should be passed in as a Child, rather than be within this component.  
We already have the `children` prop.

Reply:
A consequence of the change would be that other components that use `psammead-media-indicator` would need to be updated. 

I’ve looked at Psammead and Simorgh to see how much work this would be.
Psammead:
Stories and tests/test helpers in these two components, that can be carried out in one PR
```
packages/components/psammead-grid/src/index.stories.jsx:
packages/components/psammead-story-promo/src/index.stories.jsx:
packages/components/psammead-story-promo/src/index.test.jsx:
packages/components/psammead-story-promo/testHelpers/IndexAlsosContainer.jsx:
```
(Note, the top-level dependency in psammead would need to be updated)

In Simorgh, the changes would be to these containers - being carried out as part of the Desktop redesign: https://github.com/bbc/simorgh/pull/5139
```
src/app/containers/StoryPromo/IndexAlsos/index.jsx:
src/app/containers/StoryPromo/MediaIndicator/index.jsx:
```